lib/screens/home/local_widget/province_card.dart

```diff
@@ -0,0 +1,238 @@
+      trailing: Icon(Icons.keyboard_arrow_right),
+      children: List.generate(
+        district.length,
+        (index) {
+          String sangkat = NumberHelper.toKhmer(district[index].sangkat) + 'សង្កាត់';
+          String commune = NumberHelper.toKhmer(district[index].commune) + 'ឃុំ';
+          String village = NumberHelper.toKhmer(district[index].village) + 'ភូមិ';
+          bool isKrong = district[index].type == 'KRONG';
+          return Column(
+            children: [
+              Divider(height: 0),
+              (isKrong || isKhan)
+                  ? ListTile(
+                      title: Text((isKrong ? 'ក្រុង' : 'ខណ្ឌ') + (district[index].khmer ?? '')),
+                      subtitle: Text('$sangkat និង $village'),
+                      onTap: () {
+                        Navigator.restorablePushNamed(context, RouteConfig.DISTRICT);
+                      },
+                    )
+                  : ListTile(
+                      title: Text('ស្រុក' + (district[index].khmer ?? '')),
+                      subtitle: Text('$commune និង $villageភូមិ'),
+                      onTap: () {
+                        Navigator.restorablePushNamed(context, RouteConfig.DISTRICT);
+                      },
+                    ),
+            ],
+          );
+        },
+      ),
+    );
+  }
+
+  ListTile buildTourPlaceListTile({
+    required BuildContext context,
+    String? title,
+    String? subtitle,
+    void Function()? onTap,
+  }) {
+    return ListTile(
+      title: Text(
+        title ?? '',
+        style: Theme.of(context)
+            .textTheme
+            .headline6
+            ?.copyWith(fontWeight: FontWeight.bold, color: Theme.of(context).primaryColor),
+      ),
+      subtitle: subtitle != null
+          ? Text(
+              subtitle,
+              style: Theme.of(context).textTheme.caption,
+            )
+          : null,
+      trailing: Icon(
+        Icons.keyboard_arrow_right,
+        color: Colors.black,
+      ),
+      onTap: onTap,
+    );
+  }
+
+  ListView buildInfoCount(BuildContext context) {
+    List<String> titles = [];
+    List<String> subtitles = [];
+    if (province.srok != 0) {
+      titles.add('ស្រុក');
+      subtitles.add(NumberHelper.toKhmer(province.srok));
+    }
+    if (province.khan != 0) {
+      titles.add('ខណ្ឌ');
+      subtitles.add(NumberHelper.toKhmer(province.khan));
+    }
+    if (province.sangkat != 0) {
+      titles.add('សង្កាត់');
+      subtitles.add(NumberHelper.toKhmer(province.sangkat));
+    }
+    if (province.commune != 0) {
+      titles.add('ឃុំ');
+      subtitles.add(NumberHelper.toKhmer(province.commune));
+    }
+    if (province.village != 0) {
+      titles.add('ភូមិ');
+      subtitles.add(NumberHelper.toKhmer(province.village));
+    }
+    return ListView(
+      scrollDirection: Axis.horizontal,
+      children: List.generate(
+        titles.length,
+        (index) {
+          return Container(
+            width: 97.5,
```

Comment:
why we need width here

Reply:
Follow design

Reply:
for width, we usually follow the child inside, so it does not overflow when the font size is big.

Reply:
Child is small